src/libraries/System.DirectoryServices.Protocols/tests/DirectoryServicesProtocolsTests.cs

```diff
@@ -630,6 +630,7 @@ private LdapConnection GetConnection()
             // Set server protocol before bind; OpenLDAP servers default
             // to LDAP v2, which we do not support, and will return LDAP_PROTOCOL_ERROR
             connection.SessionOptions.ProtocolVersion = 3;
+            connection.SessionOptions.SecureSocketLayer = LdapConfiguration.Configuration.UseTls;
```

Comment:
Does it matter that this will apply to all tests?

Reply:
I don't think so? I think it just means that the `SecureSocketLayer` setter gets called twice with `false` (once in the constructor and again here) if `<UseTls />` is not set or is `false` in `LdapConfiguration.xml`. As long as that setter is idempotent, I don't think it causes any harm.

I can put a guard in though if that's safer.

Reply:
No, I think it's fine.